### PR TITLE
Dynamo stride dim kwargs

### LINKED ISSUE #97441: Dynamo: Tensor.stride() unhandled kwargs on 'dim'
_Originally posted by @ngimel in https://github.com/pytorch/pytorch/issues/97098#issuecomment-1479871997_
            

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### PR DESCRIPTION
**Pull request by nima10khodaveisi**
Fixes #97441

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Comment by nima10khodaveisi** [comment]
@ngimel Would you please review this?

File: torch/_dynamo/variables/tensor.py [hunk 1]
```diff
@@ -248,6 +248,12 @@ def call_method(
         options = VariableTracker.propagate(self, args, kwargs.values())
         if name == "stride" and self.stride is not None:
             constant_result = ConstantVariable(self.stride, **options)
+
+            if "dim" in kwargs:
+                dim = kwargs.pop("dim")
+                constant_result = constant_result.call_method(
+                    tx, "__getitem__", [dim], {}
```
Comment (ngimel) [inline torch/_dynamo/variables/tensor.py:255]: can you do `getitem_const` here?

**Comment by nima10khodaveisi** [comment]
@ezyang Would you please merge this?

**Review by ezyang — APPROVED** [review]

**Comment by nima10khodaveisi** [comment]
I think you should set label ```topic: not user facing``` 
@ezyang

**Pull request closed without merging**